lib/createWrappedExpectProto.js

```diff
@@ -112,9 +112,6 @@ module.exports = function createWrappedExpectProto(unexpected) {
                 if (e && e._isUnexpected) {
                     var wrappedError = new UnexpectedError(that, e);
                     wrappedError.originalError = e.originalError;
-                    if (that._context.serializeErrorsFromWrappedExpect) {
```

Comment:
@papandreou:

We will loose a very crazy special case:

```js
expect.addAssertion('<number> to be crazy <function>', (expect, subject, done) => {
  setTimeout(() => {
    expect(subject, 'to be',42)
    done()
  }, 10)
})

it('this will not work', (done) => {
  expect(24, 'to be crazy', done)
})
```